ckanext/switzerland/helpers/frontend_helpers.py

```diff
@@ -350,3 +350,12 @@ def render_publisher(publisher_value):
         return publisher_value
     else:
         return publisher
+
+
+def get_permalink(identifier):
+    """Given the identifier of a dataset, construct its permalink.
+    """
+    return "%s/perma/%s" % (
+        tk.config.get('ckanext.switzerland.frontend_url', ''),
```

Comment:
I would not use an empty string as a default in this case, since we do not expect our permalink to start like that